### PR TITLE
Clean up around here

### PR DESCRIPTION
**Pull request by cnunciato**
This change:

* Removes a bunch of dead code
* Removes fake-api and fixtures as we no longer need them
* Removes a bunch of unused images
* Removes all unused variable references (and adds a lint rule for that)
* Pulls the api clients into a directory of their own and renames them
* Reduces the duplication in actions/index
* Silences the Sass warnings in the Material font setup
* Removes some known-unused CSS
* Removes a long-outdated doc explaining which files do what

Signed-off-by: Christian Nunciato <cnunciato@chef.io>

tenor-221041469

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/builder-web/app/actions/index.ts [hunk 1]
```diff
@@ -12,196 +12,190 @@
...
-export const attemptSignIn = usersActions.attemptSignIn;
-export const toggleUserNavMenu = usersActions.toggleUserNavMenu;
-export const signOut = usersActions.signOut;
-
-export const setLayout = uiActions.setLayout;
-
-export const getCookie = cookieActions.getCookie;
-export const setCookie = cookieActions.setCookie;
-export const removeCookie = cookieActions.removeCookie;
-
```
Comment (raskchanky) [inline components/builder-web/app/actions/index.ts]: Just ... wow.  So good.

**Comment by raskchanky** [comment]
Reviewing this PR brought me so much joy.

tenor-74806656

**Comment by raskchanky** [comment]
tenor-191798350

**Comment by reset** [comment]
tenor-173675015

**Pull request merged**